agent_config.go

```diff
@@ -157,12 +157,6 @@ func (config *AgentConfig) initWithDefaults(a *Agent) {
 		a.maxBindingRequests = *config.MaxBindingRequests
 	}
 
-	if config.CandidateSelectionTimeout == nil {
-		a.candidateSelectionTimeout = defaultCandidateSelectionTimeout
-	} else {
-		a.candidateSelectionTimeout = *config.CandidateSelectionTimeout
```

Comment:
Can `defaultCandidateSelectionTimeout` be remove?
`AgentConfig.CandidateSelectionTimeout` would be better to be commented as deprecated.

Reply:
Since we did the API break I think it is ok to kill it!

My concern is that it does nothing anymore :( so if a user is still using it they are going to be surprised it is a no-op I think.

Reply:
I agree to just remove it from v2 API.

Reply:
So,
https://github.com/pion/ice/blob/245c513fed1c946cb5b085fab0e424b28651cce3/agent_config.go#L107-L110 
can be removed.

Reply:
done!